curation-migration-datasync/externalCaller/parserCaller.ts

```diff
@@ -0,0 +1,30 @@
+import config from '../config';
```

Comment:
moved this to seperate file coz spy doesn't work if the spied function and calling function is in the same file

Reply:
interesting...

Reply:
Yes, this is because the dependency is already 'resolved' by the imported module before you stub it

Reply:
The other way you can get around it is by wrapping in an object

Reply:
ah good idea, it worked - thanks!